src/OpenLoco/Paint/Paint.cpp

```diff
@@ -498,8 +498,8 @@ namespace OpenLoco::Paint
     static bool isPSSpriteTypeInFilter(const InteractionItem spriteType, uint32_t filter)
     {
         constexpr uint32_t interactionItemToFilter[] = { 0, 1 << 0, 1 << 0, 1 << 1, 1 << 2, 1 << 14, 1 << 7, 1 << 11, 1 << 11, 1 << 11, 1 << 11, 1 << 3, 1 << 4, 1 << 8, 1 << 12, 1 << 13, 1 << 5, 1 << 6, 000, 1 << 15, 1 << 16, 1 << 9 };
```

Comment:
use the flags here in this array

Reply:
What about the 0 and 000? I just left both as simply 0.

Reply:
Yes leave as 0. The 000 was just to draw attention to it